Bowling/Frame.cs

```diff
@@ -0,0 +1,77 @@
+﻿namespace Game
+{
+    public class Frame
+    {
+        private int firstShot;
```

Comment:
Make the variable names all in uppercase.
There is a good practice to name the bool values prefixed with `isSomething` or `areSomething`